app/javascript/channels/collect_submissions_channel.js

```diff
@@ -0,0 +1,19 @@
+import consumer from "./consumer";
+
+function create_collect_submissions_channel_subscription() {
+  consumer.subscriptions.create("CollectSubmissionsChannel", {
+    connected() {
+      // Called when the subscription is ready for use on the server
+    },
+
+    disconnected() {
+      // Called when the subscription has been terminated by the server
+    },
+
+    received(data) {
+      // Called when there's incoming data on the websocket for this channel
+      window.submissionTable.wrapped.fetchData();
```

Comment:
Oh sorry I forgot to comment on this last time, but we should avoid using global variables (from `window`) when possible. In this case, it is possible to pass in the component itself (`this`) to `create_collect_submissions_channel_subscription`, so that here you can do something like `component.fetchData()`

Reply:
Actually, please delete this file entirely, and instead convert this function into a method of the component itself.